dependency_support/com_github_openroad/bundled.BUILD.bazel

```diff
@@ -0,0 +1,1732 @@
+    srcs = [
+        "src/OpenDB/src/swig/common/swig_common.cpp",
+        "src/OpenDB/src/swig/common/swig_common.h",
+    ],
+    visibility = ["@com_github_openroad//:__subpackages__"],
+)
+
+tcl_wrap_cc(
+    name = "opendb_tcl",
+    srcs = glob([
+        "src/OpenDB/src/swig/tcl/*.i",
+        "src/OpenDB/src/swig/common/*.i",
+        "src/OpenDB/include/opendb/*.h",
+    ]),
+    module = "opendbtcl",
+    namespace_prefix = "odb",
+    root_swig_src = "src/OpenDB/src/swig/common/opendb.i",
+    swig_includes = [
+        "src/OpenDB/include/opendb",
+        "src/OpenDB/src/swig/tcl",
+    ],
+    swig_options = [
+        # These values are derived from the CMakeList.txt and represent the "rules" this swig file
+        # breaks. Swig refuses to compile our swig files unless we acknowledge we are ignoring the
+        # following warnings. They can be derived by attemtpting to compile without them and fixing
+        # the warnings one by one.
+        "-w509,503,501,472,467,402,401,317,325,378,383,389,365,362,314,258,240,203,201",
+    ],
+)
+
+# genrule(
+#     name = "py_swig",
+#     srcs = glob([
+#         "src/OpenDB/src/swig/python/*.i",
+#         "src/OpenDB/src/swig/common/*.i",
+#         "src/OpenDB/include/opendb/*.h",
+#     ]),
+#     outs = [
+#         "opendbpy.py",
+#         "opendbpy_wrap.cpp",
+#     ],
+#     cmd = " ".join([
+#         "$(location @net_sourceforge_prdownloads_swig//:swig)",
+#         "-python",
+#         "-c++",
+#         "-Ithird_party/open_road/src/OpenDB/include/opendb",
+#         "-Ithird_party/open_road/src/OpenDB/src/swig/python",
+#         "-module opendbpy",
+#         "-w551,509,503,501,472,467,402,401,317,325,378,383,389,365,362,314,258,240,203,201",
+#         "-o $(@D)/opendbpy_wrap.cpp",
+#         "-outdir $(@D)",
+#         "$(rootpath src/OpenDB/src/swig/common/opendb.i)",
+#     ]),
+#     exec_tools = [
+#         "@net_sourceforge_prdownloads_swig//:swig",
+#         "@net_sourceforge_prdownloads_swig//:swig_templates",
+#     ],
+# )
+
+# py_extension(
+#     name = "_opendbpy",
+#     srcs = [
+#         "src/OpenDB/src/swig/common/swig_common.cpp",
+#         "src/OpenDB/src/swig/common/swig_common.h",
+#         ":opendbpy_wrap.cpp",
+#     ],
+#     copts = [
+#         "-fexceptions",
+#         "-Wno-error",
+#     ],
+#     features = ["-use_header_modules"],
+#     includes = [
+#         "src/OpenDB/src/swig/common",
+#     ],
+#     deps = [
+#         ":opendb_lib",
+#         "//third_party/open_road:logger",
+#         "//third_party/python_runtime:headers",
+#     ],
+# )
+
+# genrule(
+#     name = "gen_init_py",
+#     srcs = ["opendbpy.py"],
+#     outs = ["__init__.py"],
+#     cmd = "cp $< $@",
+# )
+
+# py_library(
+#     name = "opendbpy",
+#     srcs = [
+#         ":__init__.py",
+#     ],
+#     visibility = [
+#         "//third_party/py/opendbpy:__subpackages__",
+#     ],
+#     deps = [
+#         ":_opendbpy",
+#     ],
+# )
```

Comment:
Why is this chunk commented out? Without a comment that explains why I think it's better and less confusing to just remove this code.